librz/core/cmd_descs/cmd_print.yaml

```diff
@@ -3,6 +3,25 @@
 ---
 name: cmd_print
 commands:
+  - name: pb
+    summary: Print bitstream of <n> bits, skipping the first <skip> bits.
```

Comment:
cant we just have this via `@ <num>` and `@! <num>` @ret2libc

Reply:
I don't like it too much because we are talking about bits here